pulsar-websocket/src/main/java/org/apache/pulsar/websocket/data/ConsumerMessage.java

```diff
@@ -35,4 +35,5 @@ public class ConsumerMessage {
     public EncryptionContext encryptionContext;
 
     public String key;
+    // TODO support ackSet in the future.
```

Comment:
We can remove it first. Not sure if it is necessary to support ackSet.

Reply:
it was removed